coordinator/provider/coordinator.go

```diff
@@ -406,8 +406,38 @@ func (qc *qdbCoordinator) AddRouter(ctx context.Context, router *topology.Router
 	return qc.db.AddRouter(ctx, topology.RouterToDB(router))
 }
 
-func (qc *qdbCoordinator) ListShardingRules(ctx context.Context) ([]*shrule.ShardingRule, error) {
-	rulesList, err := qc.db.ListShardingRules(ctx)
+func (qc *qdbCoordinator) getAllListShardingRules(ctx context.Context) ([]*shrule.ShardingRule, error) {
```

Comment:
We shouldn't use getAllListShardingRules because its just nonsense information without specified dataspace. What if we'll use "default" dataspace everywhere if there is no specified the opposite?

Well, ListAllKeyRanges seems the same thing